### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by NasyTheSlayer**
[DEMO LINK](https://NasyTheSlayer.github.io/layout_landing-page/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,485 @@
...
+              <img
+                class="logo__image"
+                src="images/icons/MyBiKE.svg"
+                alt="MyBiKE"
+              >
+            </a>
+            <a
+              href="tel:+1 234 5555-55-55"
+              class="icon icon--phone header__call"
+            ></a>
```
Comment (anastasiiavorobiova) [inline src/index.html:33]:
```suggestion
            >
            </a>
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,485 @@
...
+
+            <a
+              href="#menu"
+              class="icon icon--menu header__menu-opener"
+            ></a>
```
Comment (anastasiiavorobiova) [inline src/index.html:38]:
```suggestion
            >
            </a>
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,485 @@
...
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="images/icons/MyBikeBlack.svg"
+                alt="MyBiKE"
+              >
+              <a
+              href="#"
+              class="icon icon--cross"
+            ></a>
```
Comment (anastasiiavorobiova) [inline src/index.html:60]:
```suggestion
            >
            </a>
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Consider adding DEMO link to the PR description
Screenshot 2023-08-27 at 17 03 56
Consider fixing validation errors
Screenshot 2023-08-27 at 17 02 07
Consider fixing autocomplete styling (the blue background)
Screenshot 2023-08-27 at 17 01 59
Consider removing default borders

**Comment by NasyTheSlayer** [comment]
image
Didnt understand how to fix it

File: README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://<NasyTheSlayer>.github.io/layout_landing-page/)
```
Comment (volodymyr-soltys97) [inline README.md:14]:
```suggestion
- Deploy and create a Pull Request with a [DEMO LINK](https://NasyTheSlayer.github.io/layout_landing-page/)
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Demo link in the PR description, you need to fix it
image

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome work!

**Pull request closed without merging**